<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IConfigCustomize</class>
 <widget class="QWidget" name="IConfigCustomize">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="4" column="1" colspan="3">
        <widget class="QLineEdit" name="edBackground"/>
       </item>
       <item row="7" column="3">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>286</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="chkBackground">
         <property name="text">
          <string>Background image</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="chkShowWhois">
         <property name="text">
          <string>Show /whois in active window</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Quit message</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="chkReJoin">
         <property name="text">
          <string>Auto re-join channels on connect</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="chkTrayNotify">
         <property name="text">
          <string>Tray notify delay:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="chkShowMotd">
         <property name="text">
          <string>Show MOTD in own dialog</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="chkShowOptions">
         <property name="text">
          <string>Show options upon start</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="edTray">
         <property name="suffix">
          <string>s</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="btnBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2" colspan="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>371</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4" colspan="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>136</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="6">
        <widget class="QLineEdit" name="edQuit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" rowspan="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rdDefault">
            <property name="text">
             <string>Default text</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QRadioButton" name="rdLinks">
            <property name="text">
             <string>Links</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rdLocalInfo">
            <property name="text">
             <string>Local info</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="rdNotice">
            <property name="text">
             <string>Notice</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QRadioButton" name="rdInputText">
            <property name="text">
             <string>Input text</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QRadioButton" name="rdOwn">
            <property name="text">
             <string>Own text</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QRadioButton" name="rdBackground">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QRadioButton" name="rdWindowTreeText">
            <property name="text">
             <string>Windowtree text</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="rdCtcp">
            <property name="text">
             <string>CTCP</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rdActions">
            <property name="text">
             <string>Actions</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QRadioButton" name="rdNicklistText">
            <property name="text">
             <string>Nicklist text</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QRadioButton" name="rdInputBg">
            <property name="text">
             <string>Input bg.</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QRadioButton" name="rdWindowTreeBg">
            <property name="text">
             <string>Windowtree bg.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rdServerInfo">
            <property name="text">
             <string>Server info</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QRadioButton" name="rdNicklistBg">
            <property name="text">
             <string>Nicklist bg.</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QRadioButton" name="rdHighlight">
            <property name="text">
             <string>Highlight</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="1">
           <widget class="QWidget" name="colPreview" native="true">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGraphicsView" name="colorPick">
            <property name="minimumSize">
             <size>
              <width>310</width>
              <height>330</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>310</width>
              <height>330</height>
             </size>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>CrossCursor</cursorShape>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="colorCode">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>R</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>G</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>B</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>H</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>S</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>V</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="slideR">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSlider" name="slideG">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSlider" name="slideB">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QSlider" name="slideH">
         <property name="maximum">
          <number>359</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QSlider" name="slideS">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QSlider" name="slideV">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Text appearance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>186</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QFontComboBox" name="fontComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="chkTimestamp">
         <property name="text">
          <string>Display timestamp</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>219</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="edTimeFormat"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chkModes">
         <property name="text">
          <string>Show modes in messages</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="chkUnderlineLinks">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Underlined links</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkShowOptions</tabstop>
  <tabstop>chkShowWhois</tabstop>
  <tabstop>chkShowMotd</tabstop>
  <tabstop>chkTrayNotify</tabstop>
  <tabstop>edTray</tabstop>
  <tabstop>chkReJoin</tabstop>
  <tabstop>chkBackground</tabstop>
  <tabstop>edBackground</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>edQuit</tabstop>
  <tabstop>rdDefault</tabstop>
  <tabstop>rdLocalInfo</tabstop>
  <tabstop>rdServerInfo</tabstop>
  <tabstop>rdActions</tabstop>
  <tabstop>rdCtcp</tabstop>
  <tabstop>rdNotice</tabstop>
  <tabstop>rdLinks</tabstop>
  <tabstop>rdOwn</tabstop>
  <tabstop>rdHighlight</tabstop>
  <tabstop>rdBackground</tabstop>
  <tabstop>rdInputBg</tabstop>
  <tabstop>rdInputText</tabstop>
  <tabstop>rdNicklistBg</tabstop>
  <tabstop>rdNicklistText</tabstop>
  <tabstop>rdWindowTreeBg</tabstop>
  <tabstop>rdWindowTreeText</tabstop>
  <tabstop>colorCode</tabstop>
  <tabstop>slideR</tabstop>
  <tabstop>slideG</tabstop>
  <tabstop>slideB</tabstop>
  <tabstop>slideH</tabstop>
  <tabstop>slideS</tabstop>
  <tabstop>slideV</tabstop>
  <tabstop>chkModes</tabstop>
  <tabstop>chkUnderlineLinks</tabstop>
  <tabstop>chkTimestamp</tabstop>
  <tabstop>edTimeFormat</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>colorPick</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkTimestamp</sender>
   <signal>toggled(bool)</signal>
   <receiver>edTimeFormat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
